Dockerfile

```diff
@@ -17,12 +19,149 @@ COPY --from=builder \
     /tmp/bitgo/modules/express/RELEASE_NOTES.md \
     /tmp/bitgo/modules/express/package.json \
     /var/bitgo-express/
-# copy both the root node_modules and the express module node_modules to the final node_modules directory
-COPY --from=builder /tmp/bitgo/node_modules /tmp/bitgo/modules/express/node_modules /var/bitgo-express/node_modules/
-# copy individual directories that contain js executable files, source and compiled js
+
+# copy the root node_modules to the bitgo-express parent node_modules
+COPY --from=builder /tmp/bitgo/node_modules  /var/node_modules/
+
+# copy the express module node_modules to the final node_modules directory
+COPY --from=builder /tmp/bitgo/modules/express/node_modules /var/bitgo-express/node_modules/
+
+# copy individual directories that contain js executable files, and compiled js
 COPY --from=builder /tmp/bitgo/modules/express/bin /var/bitgo-express/bin/
 COPY --from=builder /tmp/bitgo/modules/express/dist /var/bitgo-express/dist/
-COPY --from=builder /tmp/bitgo/modules/express/src /var/bitgo-express/src/
+
+# copy bitgo and yarn link it
+COPY --from=builder /tmp/bitgo/modules/bitgo/package.json /var/modules/bitgo/
+COPY --from=builder /tmp/bitgo/modules/bitgo/dist /var/modules/bitgo/dist/
```

Comment:
Should we be pulling from npm as published dependencies and not internal packages?

Reply:
Based on this [ticket](https://bitgoinc.atlassian.net/browse/BG-50244)'s description, it needs to use yarn and internal packages if I understand it correctly.